### ISSUE TITLE
Imperfect RF spoiling correction: make diffusion optional

### ISSUE THREAD

**Issue by mfcallaghan** [hMRI-group/hMRI-toolbox#64]
The diffusion effect could be excluded by setting the gradient amplitudes to zero.  However, a more user friendly approach would be to add a boolean and only require input of gradient amplitudes and timings if diffusion is to be incorporated.  This would match Preibisch et al. 2009.

**Comment by lukeje** [comment]
Closing this as I think no-one is likely to work on it. Feel free to re-open if there is interest.

**Comment by Barisevrenugur** [comment]
hi @lukeje , could you create a tag something like `closed without prejudice` , hinting it has been closed with a type of comment like `Closing this as I think no-one is likely to work on it. Feel free to re-open if there is interest.`  and add it to the ones you closed like this today? so that we have a way to filter them if we need to.  thank you!

**Comment by lukeje** [comment]
I'd prefer to just wait to see if anyone ever brings up the issue again. If we get a duplicate, then that's a sure sign that it matters enough to spend time on it.

**Comment by Barisevrenugur** [comment]
the two you closed like that has `Task` which is appropriate, then maybe add tag `Task` if you need to close another one like that?  just to make filtering easier if ever needed (e.g. there is a new issue and you suspect something like that was closed before...)

**Comment by lukeje** [comment]
Thinking about it again, I guess I shouldn't just close these issues without discussion.

**Comment by Barisevrenugur** [comment]
@lukeje To me, as I have mentioned above, it is completely OK to close issues that has not been touched for while- I would not imagine anyone would have a reason to object to that. However, the only thing is, it would be really helpful if we create a tag like `closed without prejudice` or something like that so that we can filter the ones we closed like this and easily find if we ever need to re open them for some reason.  I do not think this is rocket science:  should I create a tag like that, tag the ones you closed yesterday and close them?

**Comment by lukeje** [comment]
My two cents after thinking about this more are that we shouldn't provide an easy way for people to disable diffusion spoiling. It's [been](https://doi.org/10.1002/mrm.22394) [well](https://doi.org/10.1002/mrm.28720) [demonstrated](https://doi.org/10.1002/mrm.70443) that accurate modelling of diffusion is required in order to accurately simulate the GRE signal. While I understand that finding out the diffusion spoiling parameters is difficult, computing imperfect spoiling parameters without including diffusion spoiling could lead to a false sense of security where people believe they have corrected for imperfect spoiling bias but have really just added bias in the opposite direction.

**Comment by lukeje** [comment]
@Barisevrenugur I don't think this is the appropriate forum to have this sort of meta discussion. We could either (a) move it to the developer mailing list, (b) add it to the agenda for the next developer meeting or (c) create a new issue where we try and gauge people's opinions.

**Comment by Barisevrenugur** [comment]
yes, completely agree that PRs or Issues are not for meta discussions, I mentioned adding a tag above because an issue was closed without any update, just because sitting there for a while.  at the moment, I just suggest that we leave it as is:  there are only two such issues and it is not a big problem if they remain open.  If we get a very large number issues like that we can discuss it in one of these channels

**Comment by Barisevrenugur** [comment]
with your agreement, I would like to remove (including this one) all meta discussions from this PR not to clutter the space, so that only code discussions will remain

**Comment by lukeje** [comment]
@Barisevrenugur I think I managed to hide the off topic comments... (I will also do the same to this one ;-) )
